### PR TITLE
Fix/14485 wrong entry in table self report submissions

### PR DESCRIPTION
**Pull request by 30mar**
## Description
fix wrong entry in table self report submissions

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14485

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/TestTypeSelection/SRSTestTypeSelectionViewModel.swift [hunk 1]
```diff
@@ -102,9 +102,13 @@ fileprivate extension SRSSubmissionType {
 	var optionTitle: String {
 		switch self {
 		case .srsSelfTest:
-			return AppStrings.ExposureSubmission.SRSTestTypeSelection.optionSRSSelfTestTitle
+			// This case "srsSelfTest" which is not shown in the test selection screen
+			// so we dont need a string to show on the screen
```
Comment (puneetmahali1) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/TestTypeSelection/SRSTestTypeSelectionViewModel.swift]:
```suggestion
			// so we don't need a string to show on the screen
```

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/TestTypeSelection/SRSTestTypeSelectionViewModel.swift [hunk 2]
```diff
@@ -119,7 +123,11 @@ fileprivate extension SRSSubmissionType {
 	var optionAccessibilityIdentifier: String? {
 		switch self {
 		case .srsSelfTest:
-			return AccessibilityIdentifiers.ExposureSubmission.SRSTestTypeSelection.optionSRSSelfTest
+			// This case "srsSelfTest" which is not shown in the test selection screen
+			// so we dont need a string to show on the screen
```
Comment (puneetmahali1) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/TestTypeSelection/SRSTestTypeSelectionViewModel.swift]:
```suggestion
			// so we don't need a string to show on the screen
```

**Review by puneetmahali1 — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/TestTypeSelection/SRSTestTypeSelectionViewModel.swift [hunk 3]
```diff
@@ -119,7 +123,11 @@ fileprivate extension SRSSubmissionType {
 	var optionAccessibilityIdentifier: String? {
 		switch self {
 		case .srsSelfTest:
-			return AccessibilityIdentifiers.ExposureSubmission.SRSTestTypeSelection.optionSRSSelfTest
+			// This case "srsSelfTest" which is not shown in the test selection screen
+			// so we don't need a string to show on the screen
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/TestTypeSelection/SRSTestTypeSelectionViewModel.swift]:
```suggestion
			// so we don't need an accessibilityIdentifier
```

**Review by naveeddotio — APPROVED** [review]

**Review by flxschmidt — COMMENTED** [review]
LGTM. Please check the 

- `test_When_IsSelfTestTypePreselected_True_Then_selectedSubmissionTypeShouldBeSRSSelfTest` and 
- `test_When_IsSelfTestTypePreselected_False_Then_selectedSubmissionTypeShouldBeNil` 
 
if thay can be removed ;)

**Pull request merged**